templates/foreman.socket-overrides.erb

```diff
@@ -1,3 +1,4 @@
 [Socket]
 ListenStream=
 ListenStream=<%= @listen_socket %>
+User=httpd
```

Comment:
This needs to be a variable. On Debian the user is different. You can use `$apache::user` for that value. I also think this should be behind an if and only do so when Apache is configured. We can also narrow down the permissions.
```suggestion
<% if scope['foreman::apache'] -%>
User=<%=scope['apache::user'] %>
Mode=0700
<% end -%>
```